### DISCUSSION TITLE
Is there any example on how to use it with a GraphQL Schema Registry?

### DISCUSSION THREAD

**Discussion by sbilello** [nautilus/gateway#134] [Q&A] [answered]
Hi everyone!

Is there any example or documentation on how to use it with a GraphQL Schema Registry? 
I was thinking of architecture like this one. [graphql-federation-with-schema-registry](https://github.com/pipedrive/graphql-schema-registry)

Thanks!

**Comment by AlecAivazis** [comment, marked as answer]
The nautilus gateway does not rely on a schema registry like the one built by the apollo team. Instead of having a central definition, the gateway introspect the backing services to build up the final schema.
